<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>edit</class>
 <widget class="QDialog" name="edit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>510</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>CANCEL</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lastName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>320</y>
     <width>261</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="emj2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>370</y>
     <width>41</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>üìß</string>
   </property>
  </widget>
  <widget class="QLabel" name="DETAILS">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 600 11pt &quot;Segoe UI Semibold&quot; ;</string>
   </property>
   <property name="text">
    <string>EDIT CONTACT DETAILS</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="company">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>420</y>
     <width>261</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="phnNoLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="image">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>151</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 72pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>üë§ </string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="emj3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>420</y>
     <width>41</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>üè¢ </string>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>100</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="email">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>370</y>
     <width>261</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>510</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>SAVE</string>
   </property>
  </widget>
  <widget class="QLabel" name="emj1">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>319</y>
     <width>41</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>üë§</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
